client/sites-dashboard/components/sites-container.tsx

```diff
@@ -0,0 +1,139 @@
+import { ClassNames } from '@emotion/react';
+import styled from '@emotion/styled';
+import { createInterpolateElement } from '@wordpress/element';
+import { useI18n } from '@wordpress/react-i18n';
+import EmptyContent from 'calypso/components/empty-content';
+import { SiteData } from 'calypso/state/ui/selectors/site-data';
+import { SitesTable } from './sites-table';
+
+const EmptyContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	height: 600px;
+
+	.empty-content__line {
+		max-width: 650px;
+	}
+`;
+
+type SitesContainerProps = {
+	sites: SiteData[];
+	status: string;
+};
+
+export const SitesContainer = ( { sites, status }: SitesContainerProps ) => {
+	const { __ } = useI18n();
+	if ( sites.length > 0 ) {
+		return (
+			<ClassNames>
+				{ ( { css } ) => (
+					<SitesTable
+						className={ css`
+							margin-top: 32px;
+						` }
+						sites={ sites }
+					/>
+				) }
+			</ClassNames>
+		);
+	}
+
+	switch ( status ) {
+		case 'launched':
+			return (
+				<EmptyContainer>
```

Comment:
This surrounding element is useless. The `styled` function can accept a component too if it accepts the `className` property. So we can do something like:

```js
styled(EmptyContent)`
	display: flex;
	flex-direction: column;
	justify-content: center;
	align-items: center;
	height: 600px;
`
```

And remove that surrounding `EmptyContainer` 👌

Reply:
Thanks for this piece of insight 👍